tests/conftest.py

```diff
@@ -41,9 +40,6 @@
 logger = log.setup_custom_logger("root", "master")
 logging.getLogger().setLevel(logging.INFO)
 
-docker_compose_instance = "mender" + str(random.randint(0, 9999999))
-
-docker_lock = filelock.FileLock("docker_lock")
```

Comment:
Are you sure you want to remove the lock? This is still used in several places.

It was added a long time ago to fight spurious Docker errors when docker-compose was run in parallel. I'm not sure if it's relevant anymore, so I'm ok with removing it, but I think it should be done in a separate commit in that case.

Reply:
The lock is still [there](https://github.com/mendersoftware/integration/pull/771/files#diff-d577a734c45f719bc351d949a423323fR27) since the previous PR. This is only a clean-up of the unused one in `conftest.py`.

There is an change, though, and that is that each xdist thread have a different lock object. But because they all use the same file, it should work exactly as before.